test/externalTests/README.md

```diff
@@ -0,0 +1,66 @@
+## Solidity external tests
+This directory contains scripts for compiling some of the popular open-source projects using the
+current version of the compiler and running their test suites.
+
+Since projects often don't use the latest compiler, we keep a fork of each of these projects
```

Comment:
Since @hrkrshnn started it ;-):
```suggestion
Since projects often do not use the latest compiler, we keep a fork of each of these projects
```
(personally I'd say it's fine, if not nicer, to contract, but IIRC it's policy not to in the repo)

Reply:
No worries, that's actually helpful. I shouldn't have left these in. I was going to make a final run through the file to find and eradicate these but it must have slipped my mind.

My problem is that these don't feel as casual to me as they apparently do to others so I just automatically use them everywhere and have to consciously keep myself from doing so. Downside of getting most of the exposure to English from what people write on the Internet I guess :)

Reply:
Actually from what I understand they actually aren't as casual as the policy in the repo may suggest - actually I myself rather know the opposite: I had native-English coauthors on scientific papers that *made me* use contractions :-). Still since the repo does it like that, we can keep it consistent.

Reply:
Interesting. Maybe it's regional?

Contraction removed.